engine/execution/block_result.go

```diff
@@ -172,6 +172,7 @@ func (ar *BlockAttestationResult) ChunkAt(index int) *flow.Chunk {
 		len(execRes.TransactionResults()),
 		attestRes.eventCommit,
 		attestRes.endStateCommit,
+		execRes.executionSnapshot.TotalComputationUsed(),
```

Comment:
I am guessing that this is the proper `ComputationUsed` value.
In the beginning I just summed up the `ComputationUsed` from each `TransactionResult`, e.g:
```go
var totalComputationUsed uint64 = 0
for _, txResult := range execRes.TransactionResults() {
	totalComputationUsed += txResult.ComputationUsed
}
```

But then found out about the `ExecutionSnapshot`.
Let me know if this is correct :pray: